proxy/src/http/org.rs

```diff
@@ -16,830 +16,830 @@ use crate::registry;
 
 /// Prefixed filters.
 pub fn routes<R>(
-    peer: Arc<Mutex<coco::Peer>>,
-    registry: http::Shared<R>,
-    subscriptions: notification::Subscriptions,
+  peer: Arc<Mutex<coco::Peer>>,
+  registry: http::Shared<R>,
+  subscriptions: notification::Subscriptions,
```

Comment:
These and others look like miss-configured rustfmt config, we have one for the project which should be respected by editors.